web/src/beta/features/Editor/Tabs/Storytelling/StoryPageIndicator/StoryPageIndicatorItem/index.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from "@emotion/styled";
+import { FC, memo } from "react";
+
+type Props = {
+  page: number;
+  progress: number;
+  onChangePage: (page: number) => void;
+};
+const StoryPageIndicatorItem: FC<Props> = memo(({ progress, page, onChangePage }) => {
+  return <Indicator progress={progress} type="button" onClick={() => onChangePage(page)} />;
+});
+StoryPageIndicatorItem.displayName = "StoryPageIndicatorItem";
+
+export default memo(StoryPageIndicatorItem);
```

Comment:
For the same reasons you mention about `progress` not being needed to be memoized, doesn't the same apply here? memoizing this component will actually be more performance heavy than not memoizing right?